### PR TITLE
[NFC][SYCL][Joint Matrix] Added licence header to tests

### PR DESCRIPTION
**Pull request by artemrad**

### CODE REVIEW

File: sycl/test-e2e/Matrix/element_wise_all_ops_half_impl.hpp [hunk 1]
```diff
@@ -1,3 +1,11 @@
+//===--------------------- DPC++ joint_matrix------------------------------===//
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/element_wise_all_ops_half_impl.hpp:1]: please, add file names, see for example here: https://github.com/intel/llvm/blob/4b9819f0b6bf50da03ddc9987918c8d6e09ee757/sycl/test-e2e/Matrix/element_wise_abc_impl.hpp#L1

**Review by YuriPlyakhin — CHANGES_REQUESTED** [review]
need to make the file headers consistent with the coding standard https://llvm.org/docs/CodingStandards.html#file-headers

**Comment by YuriPlyakhin** [comment]
Please, resolve formatting problem. thanks.

**Review by YuriPlyakhin — APPROVED** [review]
LGTM.
Please, add [NFC] tag to the title of your PR.

**Comment by artemrad** [comment]
@intel/llvm-gatekeepers the PR is ready.

**Pull request merged**